releases/release-1.18/release-notes-draft.md

```diff
@@ -398,97 +438,51 @@
 - Wait for kubelet & kube-proxy to be ready on Windows node within 10s ([#85228](https://github.com/kubernetes/kubernetes/pull/85228), [@YangLu1031](https://github.com/YangLu1031)) [SIG Cluster Lifecycle]
-- `(*"k8s.io/client-go/rest".Request).{Do,DoRaw,Stream,Watch}` now require callers to pass a `context.Context` as an argument. The context is used for timeout and cancellation signaling and to pass supplementary information to round trippers in the wrapped transport chain. If you don't need any of this functionality, it is sufficient to pass a context created with `context.Background()` to these functions. The `(*"k8s.io/client-go/rest".Request).Context` method is removed now that all methods that execute a request accept a context directly. ([#87597](https://github.com/kubernetes/kubernetes/pull/87597), [@mikedanese](https://github.com/mikedanese)) [SIG API Machinery, Apps, Autoscaling, CLI, Cloud Provider, Cluster Lifecycle, Instrumentation, Network, Node, Scheduling, Storage and Testing]
 - `kubectl apply  -f <file> --prune -n <namespace>` should prune all resources not defined in the file in the cli specified namespace. ([#85613](https://github.com/kubernetes/kubernetes/pull/85613), [@MartinKaburu](https://github.com/MartinKaburu)) [SIG CLI]
 - `kubectl create clusterrolebinding` creates rbac.authorization.k8s.io/v1 object ([#85889](https://github.com/kubernetes/kubernetes/pull/85889), [@oke-py](https://github.com/oke-py)) [SIG CLI]
-- `kubectl diff` now returns 1 only on diff finding changes, and >1 on kubectl errors. The "exit status code 1" message as also been muted. ([#87437](https://github.com/kubernetes/kubernetes/pull/87437), [@apelisse](https://github.com/apelisse)) [SIG CLI and Testing]
\ No newline at end of file
+- `kubectl diff` now returns 1 only on diff finding changes, and >1 on kubectl errors. The "exit status code 1" message has also been muted. ([#87437](https://github.com/kubernetes/kubernetes/pull/87437), [@apelisse](https://github.com/apelisse)) [SIG CLI and Testing]
+
+## Dependencies
+
+- Update Calico to v3.8.4 [#84163](https://github.com/kubernetes/kubernetes/pull/84163)
+- Update aws-sdk-go dependency to v1.28.2 [#87253](https://github.com/kubernetes/kubernetes/pull/87253)
+- Update CNI version to v0.8.5 [#78819](https://github.com/kubernetes/kubernetes/pull/78819)
+- Update cri-tools to v1.17.0 [#86305](https://github.com/kubernetes/kubernetes/pull/86305)
+- Pause image upgraded to 3.2 in kubelet and kubeadm [#88173](https://github.com/kubernetes/kubernetes/pull/88173)
+- Update CoreDNS version to 1.6.7 in kubeadm [#86260](https://github.com/kubernetes/kubernetes/pull/86260)
+- Update golang.org/x/crypto to fix CVE-2020-9283 [#8838](https://github.com/kubernetes/kubernetes/pull/88381)
+- Update Go to 1.13.8 [#87648](https://github.com/kubernetes/kubernetes/pull/87648)
+- Update Cluster-Autoscaler to 1.18.0 [#89095](https://github.com/kubernetes/kubernetes/pull/89095)
```

Comment:
nit: Can we still have the authors linked here like in the other notes?

Reply:
I added as requested. We still have to find a way to automate this a bit. This was all hand done.